src/renderer/i18n/en/wallet.ts

```diff
@@ -80,7 +80,10 @@ const wallet: WalletMessages = {
   'wallet.upgrade.feeError': 'Upgrade fee {fee} is not covered by your balance {balance}',
   'wallet.validations.lessThen': 'Should be less then {value}',
   'wallet.validations.graterThen': 'Should be grater then {value}',
-  'wallet.validations.shouldNotBeEmpty': 'Should not be empty'
+  'wallet.validations.shouldNotBeEmpty': 'Should not be empty',
+  'wallet.ledger.verifyAddress.modal.title': 'Verify Ledger Address',
+  'wallet.ledger.verifyAddress.modal.verifyAddress': 'Verify address',
+  'wallet.ledger.verifyAddress.modal.onYourDevice': 'on your device'
```

Comment:
You can use a placeholder for `address`
```
  'wallet.ledger.verifyAddress.modal.description': 'Verify address {address} on your device',
 ```

Reply:
You will find an example in our code how to do add html tags (search for `FormattedMessage`).